edx_event_bus_redis/internal/tests/test_consumer.py

```diff
@@ -270,37 +445,58 @@ def test_redis_consumers_disabled(self, mock_create_consumer, mock_logger):
         assert not mock_create_consumer.called
         mock_logger.error.assert_called_once_with("Redis consumers not enabled, exiting.")
 
-    @patch('edx_event_bus_redis.internal.consumer.OpenEdxPublicSignal.get_signal_by_type')
-    @patch('edx_event_bus_redis.internal.consumer.RedisEventConsumer._create_consumer')
-    @patch('edx_event_bus_redis.internal.consumer.RedisEventConsumer.consume_indefinitely')
-    def test_redis_consumers_normal(self, mock_consume, mock_create_consumer, _gsbt):
+    @patch('edx_event_bus_redis.internal.consumer.OpenEdxPublicSignal.get_signal_by_type', return_value="test-signal")
+    @patch('edx_event_bus_redis.internal.consumer.RedisEventConsumer')
+    def test_redis_consumers_normal(self, mock_consumer, _):
         call_command(
             Command(),
+            topic=['test'],
+            group_id=['test_group'],
+            signal=['openedx'],
+        )
+        mock_consumer.assert_called_once_with(
             topic='test',
-            group_id='test',
-            signal='openedx',
+            group_id='test_group',
+            signal='test-signal',
+            consumer_name=None,
+            last_read_msg_id=None,
+            check_backlog=False,
         )
-        assert mock_create_consumer.called
-        assert mock_consume.called
 
-    @patch('edx_event_bus_redis.internal.consumer.OpenEdxPublicSignal.get_signal_by_type')
-    @patch('edx_event_bus_redis.internal.consumer.RedisEventConsumer._create_consumer')
-    def test_redis_consumers_with_timestamp(self, mock_reset_offsets, mock_create_consumer):
+    @patch('edx_event_bus_redis.internal.consumer.OpenEdxPublicSignal.get_signal_by_type', return_value="test-signal")
+    @patch('edx_event_bus_redis.internal.consumer.RedisEventConsumer')
+    def test_redis_consumers_with_consumer_name(self, mock_consumer, _):
```

Comment:
Should probably also test that we can pass a message id and check_backlog

Reply:
Done.